app/decorators/decidim/assemblies/admin/assemblies_controller_decorator.rb

```diff
@@ -10,10 +10,10 @@
   alias_method :original_organization_assemblies, :collection
 
   def collection
-    if current_user.admin?
-      original_organization_assemblies
-    else
+    if current_user.department_admin?
       ::Decidim::Assemblies::AssembliesWithUserRole.for(current_user)
+    else
+      original_organization_assemblies
```

Comment:
Sorry I can't get the logic change between current change and previous lines.
I think we lost the case when `current_user` is not admin neither department_admin no?

Reply:
When is an assembly admin, the previous lines don't show child assemblies. Because `current_user.admin?` is false and is treat like a department admin and this is incorrect.

Reply:
OK, then the `original_organization_assemblies` manages both admins and assembly admins because it has been added to decidim/decidim